Mathlib/Topology/ContinuousMap/Bounded/Ascoli.lean

```diff
@@ -0,0 +1,117 @@
+/-
+Copyright (c) 2018 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel, Mario Carneiro, Yury Kudryashov, Heather Macbeth
+-/
+import Mathlib.Topology.ContinuousMap.Bounded.Basic
+import Mathlib.Topology.MetricSpace.Equicontinuity
+
+/-!
+# The Arzelà–Ascoli theorem for bounded continuous functions
```

Comment:
I don't know this area well enough... but if Ascoli makes it into the filename, shouldn't Arzela as well?

Reply:
There's another (existing) file `Topology.UniformSpace.Ascoli` with no `Arzela`.

Reply:
Probably not. Like I said, I don't know this area well enough.

Reply:
If we're making a new file, it should definitely be named `ArzelaAscoli`. Probably we should change the name of the other one too, but let's not worry about that for this PR.